client/app/components/packages/rwcds-form/show.hbs

```diff
@@ -0,0 +1,249 @@
+<div class="grid-x grid-margin-x">
+  <div class="cell large-8">
+
+    <section class="form-section">
+      <h1 class="no-margin header-large">
+        <span id="project-info" class="section-anchor"></span>
+        Reasonable Worst Case Development Scenario
+      </h1>
+
+      <h2 class="section-header header-xxlarge"
+        data-test-show="dcpProjectname">
+        {{@package.rwcdsForm.dcpProjectname}}
+      </h2>
+    </section>
+
+    <section class="form-section">
+      <h2 class="section-header">
+        <span id="" class="section-anchor"></span>
+        Description
+      </h2>
+
+      <p class="grid-x">
+        <strong class="cell auto large-padding-right">
+          Has the project changed since submission of the Pre-Application Statement (PAS)?
+        </strong>
+        <span class="cell large-3">
+          {{if @package.pasForm.dcpHasprojectchangedsincesubmissionofthepas "Yes" "No"}}
+        </span>
+      </p>
+
+      <p class="ruled-top">
+        <strong>
+          Project site description:
+        </strong>
+        <br />
+        {{@package.rwcdsForm.dcpProjectsitedescription}}
+      </p>
+
+      <p class="ruled-top">
+        <strong>
+          Proposed development description:
+        </strong>
+        <br />
+        {{@package.rwcdsForm.dcpProposedprojectdevelopmentdescription}}
+      </p>
+
+      <p class="grid-x ruled-top">
+        <strong class="cell auto large-padding-right">
+          Will construction be done in multiple phases?
+        </strong>
+        <span class="cell large-3">
+          {{#if (eq @package.rwcdsForm.dcpConstructionphasing 717170000)}}
```

Comment:
Same as (2) in the comment above. we should use the optionset helper here.
```
{{optionset 'rwcdsForm' 'dcpConstructionphasing' 'label' @package.rwcdsForm.dcpConstructionphasing}}
```
This time, the optionset helper will significantly reduce line count :) ⛳